fhir-model/src/main/java/com/ibm/fhir/model/path/evaluator/FHIRPathEvaluator.java

```diff
@@ -878,6 +882,10 @@ private IllegalArgumentException unexpectedNumberOfArguments(int arity, String f
             indentLevel++;
             
             Collection<FHIRPathNode> nodes = visit(ctx.expression());
+            if (nodes.isEmpty()) {
+                // early exit
```

Comment:
indentLevel-- should go right before the return empty()

Reply:
i thought i had pushed this change, but i must have missed it in my commit.  now addressed.